device/src/esp32_run.rs

```diff
@@ -75,6 +75,9 @@ pub fn run<'a>(resources: &'a mut Resources<'a>) -> ! {
         NonceAbSlot::load_slots(nvs.split_off_front(n_nonce_sectors))
     };
 
+    // Keep some space reserved for other potential uses in the future, 8 AB slots
+    let _reserved = nvs.split_off_end(8 * 2);
```

Comment:
split it off the front before we spit off the nonces IMO.

Reply:
After the share partition and before the nonces?

Reply:
yes I think that would be good.

Reply:
force pushed pls see change